# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libappimage/hotfix-missingcstdint.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libappimage-1.0.4-5/src/libappimage/utils/hashlib.h.vanilla	2024-06-28 12:27:43.782378450 +0300
+++ libappimage-1.0.4-5/src/libappimage/utils/hashlib.h	2024-06-28 12:40:06.817335617 +0300
@@ -2,6 +2,7 @@
 
 // system
 #include <istream>
+#include <cstdint>
 
 
 
